traceview/__init__.py

```diff
@@ -221,8 +223,42 @@ def delete(self, host_id, *args, **kwargs):
           True
 
         """
+        return self.delete_host(host_id, *args, **kwargs)
+
+    def delete_host(self, host_id, *args, **kwargs):
+        """ Delete an existing host.
+
+        :param str host_id: The id of the host to delete.
```

Comment:
@dankosaur just noticing that we are mandating a `str` (and so was previously in the `delete` function) for the host id. do you think this makes sense? or should we expect an `int`?